### PR TITLE
POWR-2396-5 Enable translation for media, groups, and taxonomy terms

### PR DESCRIPTION
**Pull request by RichardDavies**

### CODE REVIEW

**Review by joshuami — CHANGES_REQUESTED** [review]
Couple of comments. I'd probably just drop the 3 configs from this PR.

File: web/sites/default/config/language.content_settings.taxonomy_term.city_section.yml [hunk 1]
```diff
@@ -8,9 +8,9 @@ dependencies:
     - content_translation
 third_party_settings:
   content_translation:
-    enabled: false
+    enabled: true
```
[the remark this answers is not in the document]
Comment (RichardDavies) [reply web/sites/default/config/language.content_settings.taxonomy_term.city_section.yml]: Should be fixed now.

File: web/sites/default/config/language.content_settings.taxonomy_term.council_agenda_item_disposition.yml [hunk 2]
```diff
@@ -15,4 +15,4 @@ id: taxonomy_term.council_agenda_item_disposition
 target_entity_type_id: taxonomy_term
 target_bundle: council_agenda_item_disposition
 default_langcode: site_default
-language_alterable: false
+language_alterable: true
```
[the remark this answers is not in the document]
Comment (RichardDavies) [reply web/sites/default/config/language.content_settings.taxonomy_term.council_agenda_item_disposition.yml]: Should be fixed now.

File: web/sites/default/config/language.content_settings.taxonomy_term.permit_type.yml [hunk 3]
```diff
@@ -8,9 +8,9 @@ dependencies:
     - content_translation
 third_party_settings:
   content_translation:
-    enabled: false
+    enabled: true
```
[the remark this answers is not in the document]
Comment (RichardDavies) [reply web/sites/default/config/language.content_settings.taxonomy_term.permit_type.yml]: Should be fixed now.

**Review by joshuami — APPROVED** [review]

**Pull request merged**